misk/src/test/kotlin/misk/hibernate/DbMovie.kt

```diff
@@ -0,0 +1,36 @@
+package misk.hibernate
+
+import java.util.Date
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType.IDENTITY
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Entity
+@Table(name = "movies")
+class DbMovie() {
+  @Id
+  @GeneratedValue(strategy = IDENTITY)
+  var id: Long? = null
+
+  var name: String? = null
```

Comment:
why are these nullable?

Reply:
makes it easier for Hibernate to instantiate. Might play with this.